assets/samples/cloudNativeFullStack.yaml

```diff
@@ -257,10 +257,10 @@ spec:
     #
     # priorityClassName: priority-class
 
-    # Optional: Adds additional annotations to the ActiveGate pods
+    # Optional: Adds custom annotations to ActiveGate pods
     #
     # annotations:
-    #   annotation-key: annotation-value
+    #   custom: annotation
```

Comment:
```suggestion
    #   custom-key: annotation
```

Reply:
IMHO customers are mature enough to know that the first part of an annotation is always the key